website/api/secure-sdlc.md

```diff
@@ -0,0 +1,74 @@
+# RISC Zero Secure Development Life-cycle
+
+RISC Zero believes in a comprehensive risk-based approach to security throughout our product and software development life-cycle.
+
+This page provides an outline of the key practices that we utilize to harden our products and services against threats. These security techniques are continuously evolving as we learn from our experiences and adapt to new threats, and we encourage you to provide feedback or suggestions for improvement.
+
+The key mechanisms that drive our process evolution are:
+* External critique from customers, partners, and the security community
+* Escape analysis of production defects and postmortem analysis of security incidents
+* Identifying new potential activities based on industry frameworks including [NIST SSDF], [ISO 27002:2022], [BSIMM 14], [the ReKT test], and [OWASP SAAAM].
+* Use of a maturity framework (known as [FACTORS]) to self-assess process maturity in the following areas:
```

Comment:
[FACTORS] needs a linik

Reply:
Synopsys had this IP at one point, but it's faded away a bit.  I think in the future we can maybe blog about it then link back to that..